lantern_cli/src/http_server/pq.rs

```diff
@@ -1,3 +1,4 @@
+use crate::http_server::index::CreateIndexInput;
```

Comment:
I think this was initially wrong, in the route instead of `CreateIndexInput` it should have been `CreatePQInput` which is declared in the file, but for the previous versions maybe macro didn't expanded correctly and it didn't fail the build

Reply:
fixed